uber/config.py

```diff
@@ -131,6 +143,16 @@ def __getattr__(self, name):
                 return sa.localized_now() > date_setting
         elif name.startswith('HAS_') and name.endswith('_ACCESS'):
             return getattr(c, name.split('_')[1]) in sa.AdminAccount.access_set()
+        elif name.endswith('_AVAILABLE'):
+            item_check = name.split('_', 1)[0]
```

Comment:
Here's a purely stylistic question for you, since you're an impartial-ish third party.  For properties and other methods on this class, ``c`` and ``self`